tensorly/metrics/tests/test_entropy.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import numpy.random as npr
+import numpy.linalg as npl
+import tensorly as tl
+from ..entropy import vonNeumann_entropy
+from ...decomposition import parafac
+
+state = npr.rand(8, 1).astype(np.float32)
```

Comment:
I would define these inside the unit-test functions. If we no longer cast to float it would also be better to use tl.randn and tl.tensor so the arrays are in created with the right backend.